__init__.py

```diff
@@ -191,8 +191,11 @@ def main():
             importer = CommandLineCraftImporter()
             result = importer.execute(bpy.context, args.input_file, args.colliders, args.allow_no_material_mesh)
         else:
-            importer = CommandLineImporter()
-            result = importer.execute(bpy.context, args.input_file, args.colliders)
+            try:
+                importer = CommandLineImporter()
+                result = importer.execute(bpy.context, args.input_file, args.colliders)
+            except EOFError:
+                print('ERROR: the .mu file is invalid')
```

Comment:
is it really possible to have an `EOFError` here? you already catch those in `import_craft.py` and do not re-raise